.azure-pipelines.yml

```diff
@@ -1,145 +1,124 @@
-      iosProvisioningProfileFile: $(InternalProvisioningProfile)
-      iosCertificateFile: $(InternalCertificate)
-      iosVariableGroup: 'ApplicationTemplate.Distribution.Internal.iOS'
-      BannerVersionNameText: "STAGING"
-
-- stage: AppCenter_TestFlight_Staging
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
-  dependsOn: Build_Staging
-  jobs:
-  - template: build/stage-release-appcenter.yml
-    parameters:
-      applicationEnvironment: Staging
-      deploymentEnvironment: AppCenter
-      appCenterWindowsSlug: $(AppCenterWindowsSlug)
-      appCenteriOSSlug: $(AppCenteriOSSlug)
-      appCenterAndroidSlug: $(AppCenterAndroidSlug)
-      androidKeyStoreFile: $(InternalKeystore)
-      androidVariableGroup: 'ApplicationTemplate.Distribution.Internal.Android'
-      appCenterServiceConnectionName: $(AppCenterServiceConnection)
-      appCenterDistributionGroup: $(AppCenterDistributionGroup)
-
-  - template: build/stage-release-appstore.yml
-    parameters:
-      applicationEnvironment: Staging
-      deploymentEnvironment: TestFlight
-
-- stage: Build_Production
-  dependsOn: Build_Staging
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
-  jobs:
-  - template: build/stage-build.yml
-    parameters:
-      applicationEnvironment: Production
-      androidKeyStoreFile: $(GooglePlayKeystore)
-      androidVariableGroup: 'ApplicationTemplate.Distribution.GooglePlay'
-      iosProvisioningProfileFile: $(AppStoreProvisioningProfile)
-      iosCertificateFile: $(AppStoreCertificate)
-      iosVariableGroup: 'ApplicationTemplate.Distribution.AppStore'
-
-- stage: AppCenter_Production
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
-  dependsOn: Build_Production
-  jobs:
-  - template: build/stage-release-appcenter.yml
-    parameters:
-      applicationEnvironment: Production
-      deploymentEnvironment: 'AppCenter Prod'
-      appCenterWindowsSlug: $(AppCenterWindowsSlug_Production)
-      appCenteriOSSlug: $(AppCenteriOSSlug_Production)
-      appCenterAndroidSlug: $(AppCenterAndroidSlug_Production)
-      androidKeyStoreFile: $(GooglePlayKeystore)
-      androidVariableGroup: 'ApplicationTemplate.Distribution.GooglePlay'
-      appCenterServiceConnectionName: $(AppCenterServiceConnection)
-      appCenterDistributionGroup: $(AppCenterDistributionGroup)
-
-- stage: AppStore
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
-  dependsOn: Build_Production
-  jobs:
-  - template: build/stage-release-appstore.yml
-    parameters:
-      applicationEnvironment: Production
-      deploymentEnvironment: AppStore
-
-- stage: GooglePlay
-  condition: and(succeeded(), eq(variables['IsLightBuild'], 'false'))
-  dependsOn: Build_Production
-  jobs:
-  - template: build/stage-release-googleplay.yml
-    parameters:
-      applicationEnvironment: Production
-      deploymentEnvironment: GooglePlay
+﻿# Use this to schedule a build at defined days and times

```

Comment:
Why does it look like every line changed?

Reply:
It was caused by VS Code "Format on save"